ADAL/ADAL.xcodeproj/project.pbxproj

```diff
@@ -1127,6 +1168,24 @@
 /* End PBXHeadersBuildPhase section */
 
 /* Begin PBXNativeTarget section */
+		60967E071D76B56D00863853 /* ADAL Preprocessing */ = {
```

Comment:
Avoid having a separate target for this as it muddles up the dependencies, it'd be better to just add the run script phase in each of the applicable target